cmd/tbcd/README.md

```diff
@@ -1,56 +1,113 @@
-# tbcd
+# 🌐 Hemi Tiny Bitcoin Daemon (`tbcd`)
 
-## Hemi Tiny Bitcoin Daemon
+`tbcd` is a **minimal Bitcoin block downloader** and **indexer daemon**. It is designed to wrap the `tbc` service, which
+provides Bitcoin data **(blocks and transactions)** for other applications. Primarily, **it functions as a network RPC
+endpoint.**
 
-tbcd is a very minimal bitcoin block downloader and indexer meant for embedding in other applications that require access to bitcoin data (blocks and txes).
+## 🖥️ System Requirements
 
-tbcd requires sufficient disk space for a full download of bitcoin block data on a fast (preferably ssd or better disk.
+`tbcd` requires significant system resources due to its functionality:
 
-tbcd is build with the heminetwork makefile,  To build standalone (requires `go 1.21+`), type:
+| Requirement   | Specification    |
+|---------------|------------------|
+| **CPU Cores** | 4 minimum        |
+| **RAM**       | 8 GiB minimum    |
+| **Disk**      | NVMe recommended |
 
-``` sh
-cd heminetowkr/cmd/tbcd
-go build
-```
+**As of April 18, 2024:**
 
-On some linux systems you may need to increase the number of open files allowed (particularly with slower disks) and the maximum stack size.  If you run into open file or OOM errors, in the shell you are going to run tbcd, run:
+- **`testnet3`** requires approximately 40 GiB of disk space.
+- **`mainnet`** requires over 300 GiB of disk space. <!-- XXX: add exact number here -->
 
-```sh
-ulimit -n 8192
-ulimit -s 8192
-```
+---
+
+## 🛠️ Building `tbcd` From Source
+
+### 🏁 Prerequisites
+
+Ensure Go v1.22 or newer is installed on your system.
```

Comment:
Same comment/question as above on the go version.

Reply:
Yep, updated.